.github/workflows/linting.yml

```diff
@@ -0,0 +1,48 @@
+name: CG installation and linting
+
+on: push
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+
+    # Check out cg code
+    - uses: actions/checkout@v1
+
+    # Set up python
+    - name: Set up Python ${{ matrix.python-version}}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version}}
+
+    - name: Production Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install .
+        pip check
+
+    - name: Development Dependencies
+      run: |
+        pip install -r requirements-dev.txt
+        pip install flake8
+        pip install black
```

Comment:
If these two are dev-reqs then they could go into the dev-reqs file, otherwise they could be skipped here?

Reply:
I do not agree. These packages are not necessary for development only for linting. I suggest that packages that are required for executing tests etc should be part of req-dev